src/ray/raylet/raylet.cc

```diff
@@ -69,17 +69,17 @@ Raylet::Raylet(boost::asio::io_service &main_service, const std::string &socket_
                                                                     gcs_client_))
               : std::dynamic_pointer_cast<ObjectDirectoryInterface>(
                     std::make_shared<ObjectDirectory>(main_service, gcs_client_))),
-      object_manager_(main_service, self_node_id_, object_manager_config,
-                      object_directory_,
-                      [this](const ObjectID &object_id, const std::string &spilled_url,
-                             std::function<void(const ray::Status &)> callback) {
-                        node_manager_.GetLocalObjectManager().AsyncRestoreSpilledObject(
-                            object_id, spilled_url, callback);
-                      },
-                      [this](int64_t num_bytes_required) {
-                        return node_manager_.GetLocalObjectManager().SpillObjectsOfSize(
-                            num_bytes_required);
-                      }),
+      object_manager_(
+          main_service, self_node_id_, object_manager_config, object_directory_,
+          [this](const ObjectID &object_id, const std::string &spilled_url,
+                 std::function<void(const ray::Status &)> callback) {
+            node_manager_.GetLocalObjectManager().AsyncRestoreSpilledObject(
+                object_id, spilled_url, callback);
+          },
+          [this](int64_t num_bytes_required) {
+            return node_manager_.GetLocalObjectManager().SpillObjectsOfSize(
+                num_bytes_required);
+          }),
```

Comment:
remove :) ?I think you may want to update your clang (or uninstall)

Reply:
Argh. Updated clang-format and removed this from PR. I'll make sure to check this next time!